Containerfile

```
@@ -0,0 +1,29 @@
+FROM quay.io/operate-first/opf-toolbox:v0.4.1 AS bins
+ADD . /scripts
+WORKDIR /scripts
+RUN file="$(ls -1 /usr/local/bin)" && echo $file
+RUN mv  /usr/local/bin /scripts/
+
+
+# s2i-custom-py38-notebook
+FROM quay.io/thoth-station/s2i-minimal-py38-notebook:v0.1.0
+MAINTAINER Anand Sanmukhani <asanmukh@redhat.com>
+LABEL io.k8s.description="Custom JupyterHub Notebook Builder Image" \
+      io.openshift.s2i.scripts-url=image:///opt/app-root/builder \
+      io.s2i.scripts-url=image:///opt/app-root/builder
+
+ENV \
+    # DEPRECATED: Use above LABEL instead, because this will be removed in future versions.
+    STI_SCRIPTS_URL=image:///opt/app-root/builder \
+    # Path to be used in other layers to place s2i scripts into
+    STI_SCRIPTS_PATH=/opt/app-root/builder
+
+# rename thoth assemble and run scripts
+RUN mv /opt/app-root/builder/assemble /opt/app-root/builder/assemble.original
+RUN mv /opt/app-root/builder/run /opt/app-root/builder/run.original
+COPY --from=bins /scripts/* /usr/local/bin/
+# Copy the builder files into /opt/app-root
+COPY ./s2i/bin/* /opt/app-root/builder/
+
+# Copy sample notebook to the image
+COPY --chown=1001:root ./notebooks /opt/app-root/backup
```

All this can be simplified into:

```suggestion
FROM quay.io/thoth-station/s2i-custom-py38-notebook:v0.3.2
```

You just want to use the image, you don't want to redefine it :slightly_smiling_face:

---

Thank you for review! Didn't know that there was this custom image online :). I have cleaned up in latest commit whole file and also fixed ksops which should now work, there is still pre-commit which I am going to have to dig up more tomorrow to get it but rest should be working (or at least it works on my machine XD).